### PR TITLE
Portal sidebar

### PR DESCRIPTION
**Pull request by andrewbiang888**
This PR is in reference to GH-299 and is part of GH-302.

### CODE REVIEW

File: app/src/Pages/Portal/Portal.js [hunk 1]
```diff
@@ -4,36 +4,72 @@ import { CSSTransition, TransitionGroup } from "react-transition-group";
...
+    width: 80%;
+    padding-left: 20px;  
+    border-left: 1px solid rgba(0, 0, 0, 0.15);
+  }
+  
+  .content-wrapper, .sidebar-wrapper {
+    float: left;
+    padding-top: 20px;  
+  }
+`;
```
Comment (jaronheard) [inline app/src/Pages/Portal/Portal.js]: Per discussion with @andrewbiang888 -- going to refactor the css in a separate PR

File: app/src/Pages/Portal/Portal.js [hunk 2]
```diff
@@ -4,36 +4,72 @@ import { CSSTransition, TransitionGroup } from "react-transition-group";
...
+            <Route
+              render={({ location }) => (
+                <>
+                  <TransitionGroup className="oe-portal-container">
+                    <CSSTransition
+                      key={location.pathname}
+                      timeout={{ enter: 500, exit: 300 }}
+                      classNames="page"
+                      appear
+                    >
```
Comment (jaronheard) [inline app/src/Pages/Portal/Portal.js]: Look further at this transition group later

File: app/src/Pages/routes.js [hunk 3]
```diff
@@ -17,8 +17,9 @@ import Portal from "./Portal/Portal";
 const Routes = props => {
   return (
     <Route
-      render={({ location }) => (
+      render={({ location, match }) => (
         <>
+          {console.log(match)}
```
Comment (jaronheard) [inline app/src/Pages/routes.js]: Unnecessary console log

**Comment by andrewbiang888** [comment]
@billoosijok How's it lookin dude?

**Comment by billoosijok** [comment]
Looking great

**Pull request merged**